spec/factories/users.rb

```diff
@@ -89,5 +93,15 @@
         full_name.downcase.gsub(' ', '.') + '@computacenter.com'
       end
     end
+
+    trait :relevant_to_computacenter do
```

Comment:
I think it would be clearer if these were replaced with multiple traits e.g `:orders_devices, :has_seen_privacy_notice` at the point of use rather than `:relevant_to_computacenter` as that doesn't tell me much - but it might just be a personal preference

Reply:
I just wanted a convenient way to factory users for my tests. I think it would be less readable if I have toggle each individual setting each time